falco/templates/NOTES.txt

```diff
@@ -23,3 +23,9 @@ Full list of outputs: https://github.com/falcosecurity/charts/tree/master/falcos
 You can enable its deployment with `--set falcosidekick.enabled=true` or in your values.yaml. 
 See: https://github.com/falcosecurity/charts/blob/master/falcosidekick/values.yaml for configuration values.
 {{- end}}
+
+{{- if and (not (empty .Values.falco.load_plugins)) (or .Values.falcoctl.artifact.follow.enabled .Values.falcoctl.artifact.install.enabled) }}
+
+WARNING:
+{{ printf "It seems you are loading the following plugins %v, please make sure to install them by adding the correct reference to falcoctl.config.artifact.install.refs: %v" .Values.falco.load_plugins .Values.falcoctl.config.artifact.install.refs -}}
+{{- end }}
```

Comment:
Not sure about this. k8saudit and cloudtrail are included in the container image, and it shouldn't be a hard requirement to use falcoctl to use them.

That should be working already since when both the _install_ and _follow_ are disabled, no empty dir is mounted.

Reply:
The warning is printed only if we are loading plugins and one of `falcoctl.artifact` or `falcoctl.install`. So yes, it does what you are saying.

Reply:
You are right!